tanjun/conversion.py

```diff
@@ -120,45 +119,35 @@ class BaseConverter(abc.ABC):
     __slots__ = ("__weakref__",)
 
     @property
-    @abc.abstractmethod
     def async_caches(self) -> collections.Sequence[typing.Any]:
-        """Collection of the asynchronous caches that this converter relies on.
-
-        This will only be necessary if the suggested intents or cache_components
-        aren't enabled for a converter which requires cache.
-        """
+        """Deprecated property."""
+        return []
```

Comment:
Could we build these returns from self.caches instead of always returning nothing here just to make it a bit more backwards compatible